src/promptflow-devkit/promptflow/_sdk/schemas/_flow.py

```diff
@@ -60,14 +61,52 @@ class FlowSchema(BaseFlowSchema):
     node_variants = fields.Dict(keys=fields.Str(), values=fields.Dict())
 
 
+class FlexFlowInputSchema(FlowInputSchema):
+    type = fields.Str(
+        required=True,
+        validate=validate.OneOf(
+            [
+                ValueType.DOUBLE,
```

Comment:
The spec is expecting we return types in a language agnostic manner. 
like double -> number

Reply:
https://json-schema.org/understanding-json-schema/reference/type

Reply:
There is no `number` in current promptflow context so UX may not recognize `number` for now. Not sure if we should ask UX to recognize both and change this for all (including dag flow)